grpc_opa/entitled_features.go

```diff
@@ -71,3 +74,97 @@ func FlattenRawEntitledFeatures(efIfc interface{}) ([]string, error) {
 
 	return result, nil
 }
+
+type opbench struct {
+	val map[string][]string
+	err error
+	log string
+}
+
+// EntitlsCtxOp is a simple operator to manipulate
+// over entitled features in context. It retrieves
+// entitled features from the context and converts
+// them to a structure that is easy to operate on.
+// Returned value includes only unimported fields
+// and is designed to chain other methods that it
+// has.
+func EntitlsCtxOp(ctx context.Context) *opbench {
```

Comment:
I think 'EntitlementsCtxOp' is a better name

Reply:
Renamed